File: app/src/main/java/com/example/edward/nyansapo/selectSchool.java
package com.example.edward.nyansapo;

import android.app.ActivityOptions;
import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class selectSchool extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_select_school);
    }

    public void openSchool(View v){
        Intent myIntent = new Intent(getBaseContext(), student.class);
        startActivity(myIntent, ActivityOptions.makeSceneTransitionAnimation(this).toBundle());
    }

    public void exportData(View v){
        Toast.makeText(getApplicationContext(), "Data has been exported successfully", Toast.LENGTH_LONG).show();
    }

    public void newSchool(View v){
        Intent myIntent = new Intent(getBaseContext(), registerSchool.class);
        startActivity(myIntent, ActivityOptions.makeSceneTransitionAnimation(this).toBundle());
    }

    public void selectFilter(View v){
        Toast.makeText(getApplicationContext(), "Selected Filter", Toast.LENGTH_LONG).show();
    }

    public void selectSort(View v){
        Toast.makeText(getApplicationContext(), "Selected Sort", Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/edward/nyansapo/selectSchool.kt
package com.nyansapo

import android.os.Bundle
import com.nyansapo.R
import android.content.Intent
import com.nyansapo.student_activity
import android.app.ActivityOptions
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.nyansapo.registerSchool

class selectSchool : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_select_school)
    }

    fun openSchool(v: View?) {
        val myIntent = Intent(baseContext, student_activity::class.java)
        startActivity(myIntent, ActivityOptions.makeSceneTransitionAnimation(this).toBundle())
    }

    fun exportData(v: View?) {
        Toast.makeText(applicationContext, "Data has been exported successfully", Toast.LENGTH_LONG).show()
    }

    fun newSchool(v: View?) {
        val myIntent = Intent(baseContext, registerSchool::class.java)
        startActivity(myIntent, ActivityOptions.makeSceneTransitionAnimation(this).toBundle())
    }

    fun selectFilter(v: View?) {
        Toast.makeText(applicationContext, "Selected Filter", Toast.LENGTH_LONG).show()
    }

    fun selectSort(v: View?) {
        Toast.makeText(applicationContext, "Selected Sort", Toast.LENGTH_LONG).show()
    }
}